config/passport.js

```diff
@@ -168,6 +168,12 @@ const metisLogin = (passport, jobs, io) => {
           valid = false;
           return done(null, false, req.flash('loginMessage', 'Account is not registered'));
         }
+
+        if (!user.validEncryptionPassword(containedDatabase.encryptionPassword)) {
+          valid = false;
+          return done(true, null, req.flash('loginMessage', 'Wrong encryption password'));
```

Comment:
Would be good to abstract `req.flash` to an agnostic function for web and mobile.  Web, use `req.flash(whatIsThis, message)`, mobile do something like `res.send({error: true, message});`

req.flash used for a LOT of stuff in this repo - need to surface it to mobile.

Reply:
Ok, let me fix it, mobile app does need it